python/ml4ir/base/features/feature_layer.py

```diff
@@ -8,6 +8,9 @@
 from ml4ir.base.features.feature_fns.categorical import categorical_embedding_with_hash_buckets
 from ml4ir.base.features.feature_fns.categorical import categorical_embedding_with_indices
 from ml4ir.base.features.feature_fns.categorical import categorical_embedding_with_vocabulary_file
+from ml4ir.base.features.feature_fns.categorical import (
+    categorical_embedding_with_vocabulary_file_and_dropout,
+)
 from ml4ir.base.io.file_io import FileIO
```

Comment:
@lastmansleeping Sorry so many minor comments, but why import it this way? Do you expect this to become a list in the future?

Reply:
Oh nice. I didn't do this. I think the PEP-8 plugin we have on the repo auto-formated it. Pretty cool really.